src/lib/plugins/blocks.js

```diff
@@ -11,7 +11,7 @@ module.exports.player = function (player, serv) {
   }
 
   player.sendBlock = (position, blockType, blockData) => // Call from player.setBlock unless you want "local" fake blocks
-    player.behavior('sendBlock', {
+    player.behavior('sendblock', {
```

Comment:
Why did you change that, was it wrong ? Are the docs up to date on this ?

Reply:
Ok then it should be updated the other way round.
The correct spelling is sendBlock, see https://github.com/PrismarineJS/flying-squid/blob/master/docs/API.md#sendblock

Reply:
Fixed

Reply:
Oh wait, the command "setblock" shouldn't be changed...